ansible/tasks/server_tasks.yml

```diff
@@ -38,6 +38,11 @@
     chdir: "{{ multinet_server_path }}"
   when: multinet_git.changed
 
+- name: Copy the .env.default to .env
+  command: cp .env.default .env
```

Comment:
Can you check to see if there is a COPY module for Ansible? It would be better to use that because it will understand idempotency for copying files in a way that using `command` will not.

Reply:
Fixed in f3caef5